features/step_definitions/home_beta_page_steps.rb

```diff
@@ -8,4 +8,12 @@
 Then(/^I should be presented with popular tools and calculators$/) do
   expect(home_beta_page.popular_tools)
     .to have_content(I18n.t('home.show.tools_heading'))
-end
\ No newline at end of file
+end
+
+Then(/^I should be presented with the seasonal spotlight$/) do
+  expected_en = "Easy ways to catch the saving habit"
+  expected_cy = "Ffyrdd hawdd i ddal y arferiad cynilo"
+
+  expect(home_beta_page.seasonal_spotlight)
+    .to have_content(eval("expected_#{I18n.locale}"))
```

Comment:
Can you change eval to send, please? 
The difference is: 
**sends** Invokes the method identified by symbol, passing it any arguments specified - **which I think it is what you want**.
*eval* Evaluates a string containing Ruby source code, or the given block, within the context of the receiver (obj) - which it is more than you actually needed.

Reply:
I cant seem to get the syntax right, can we pair when you are in?